components/infra-proxy-service/server/v1/orgs.go

```diff
@@ -0,0 +1,213 @@
+package v1
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/chef/automate/api/interservice/infra_proxy/request"
+	"github.com/chef/automate/api/interservice/infra_proxy/response"
+	"github.com/chef/automate/components/infra-proxy-service/service"
+	"github.com/chef/automate/components/infra-proxy-service/storage"
+	uuid "github.com/chef/automate/lib/uuid4"
+)
+
+// CreateOrg creates a new org
+func (s *Server) CreateOrg(ctx context.Context, req *request.CreateOrg) (*response.CreateOrg, error) {
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
cant remember, what's the reason for doing this

Reply:
@jaym I think in interservice we have defined interfaces that need to be implemented by the infra proxy server.

Reply:
I think this is so that when this function is left all the processes created from this function stop.